src/search-insights.ts

```diff
@@ -341,3 +346,7 @@ async function determineCommitsToSearch(dates: Date[], repo: string): Promise<Se
 
     return commitOids
 }
+
+function getQueryWithCountFilter(query: string): string {
+    return queryHasCountFilter(query) ? query : `${query} count:99999`
```

Comment:
Curious, is it a backend issue that query errors without count argument or it's expected behaviour that's documented in the graphql schema?

Reply:
This is a good call. 
I just realized that we can use probably just count: all as default value for count filter

Here is a small description of the count filter from our search grammar doc
>Retrieve N results. By default, Sourcegraph stops searching early and returns if it finds a full page of results. This is desirable for most interactive searches. To wait for all results, use count:all.

Reply:
But I would prefer to merge this PR and move further here with search query syntax in separate story about query search input https://github.com/sourcegraph/sourcegraph/issues/21339